web/packages/teleport/src/services/websession/websession.ts

```diff
@@ -146,11 +145,14 @@ const session = {
       return false;
     }
 
-    // Renew session if token expiry time is less than 3 minutes.
+    // Renew session if token expiry time is less than renewTime (with a floor
+    // of 30s and a ceiling of 3m).
```

Comment:
I wouldn't mention the specific values of the constants here, it's likely to go out of date if we change them in the future.

Reply:
Done.